src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -85,7 +85,10 @@ internal static class DefinitionItemExtensions
             //
             // This ensures that the type will both display properly to the user, as well as ensuring
             // that we can accurately resolve the type later on when we try to navigate to it.
-            definition = definition.OriginalDefinition;
+            if (!definition.IsTupleField())
+            {
+                definition = definition.OriginalDefinition;
+            }
```

Comment:
why?  needs comment.

Reply:
Tuples and tuple fields used to always be definitions, but they no longer necessarily are. This change emulates the previous behavior (ie. pretends that they are definitions).

---
In reply to: [355810225](https://github.com/dotnet/roslyn/pull/39370#discussion_r355810225) [](ancestors = 355810225)

Reply:
Tuples and tuple fields used to always be definitions, but they no longer necessarily are. This change emulates the previous behavior (ie. pretends that they are definitions).